apps/spark-3.5/build.gradle

```diff
@@ -16,6 +16,9 @@ ext {
     tablesTestFixturesModule = ":tables-test-fixtures:tables-test-fixtures-iceberg-1.5_2.12"
 }
 
+configurations.all {
+    exclude group: 'org.apache.iceberg', module: 'iceberg-data'
```

Comment:
i tried this and 2 or 3 submodules e.g. libs:datalayout, services:common, we're also bringing in iceberg-data

this is a quick workaround. later i will try to not use this block, but it may still become more readable/understandable to have this block vs excluding iceberg-data from several sources